network-mux/test/Test/Mux.hs

```diff
@@ -259,16 +265,16 @@ instance Arbitrary ArbitrarySDU where
 
             return $ ArbitraryInvalidSDU (InvalidSDU (RemoteClockModel ts) mid len realLen p)
                                          state
-                                         MuxDecodeError
+                                         Compat.MuxDecodeError
 
-instance Arbitrary MuxBearerState where
-     -- XXX Larval and Connected test behaviour is dependant on version
+instance Arbitrary Compat.MuxBearerState where
+     -- XXX Compat.Larval and Compat.Connected test behaviour is dependant on version
      -- negotation so they are disabled for now.
-     arbitrary = elements [ -- Larval
-                            -- , Connected
-                            Mature
-                          , Dying
-                          , Dead
+     arbitrary = elements [ -- Compat.Larval
+                            -- , Compat.Connected
```

Comment:
Can we remove these commented out lines?

Reply:
Yes, there are also some remnants of the old version negotiation scheme in the `prop_demux_sdu` which should be cleaned up.